dts/bindings/arm/arm,ethos-u.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Arm Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Arm Ethos-U NPU device tree binding
+
+compatible: "arm,ethos-u"
+
+include: base.yaml
+
+properties:
+    reg:
+        required: true
+
+    interrupts:
+        required: true
+
+    secure-enable:
+        type: boolean
+        required: true
```

Comment:
Combining `required: true` with boolean doesn't really make sense. Are you sure?

Reply:
I have removed required, as you are perfectly right that it does not make any sense at all.